<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemStatistic</class>
 <widget class="QDialog" name="DialogSystemStatistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="WidgetTextLang" name="widgetName" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>178</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelScriptAbbreviation">
           <property name="text">
            <string>Script abbreviation:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditAbbrevation">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxProperties">
         <property name="title">
          <string>Properties</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButtonFix">
              <property name="text">
               <string>Fix</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>208</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2" rowspan="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>78</width>
                <height>48</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButtonBar">
              <property name="text">
               <string>Bar:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="WidgetPicture" name="widgetPictureBar" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTextLang</class>
   <extends>QWidget</extends>
   <header>widgettextlang.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemStatistic</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemStatistic</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
